tests/test_enrollment.py

```diff
@@ -51,6 +51,26 @@ def test_enroll_from_form(self):
         self.studentB['id'] = user.id
         
         self.enrollment_matches_info(user, self.studentB)
+
+    def test_unenroll(self):
+        self.setup_course()
+
+        user = User(name=self.studentA['name'], email=self.studentA['email'])
+        db.session.add(user)
+        db.session.commit()
+        self.studentA['id'] = user.id
+        
+        Enrollment.create(self.course.id, [self.studentA])
+        enrollment = Enrollment.query.filter_by(
+            course_id=self.course.id, 
+            user_id=user.id).one_or_none()
+        assert enrollment
```

Comment:
It's better to explicit about what we expect. In this case it looks like you are asserting that `enrollment` is not None and the later one you are asserting that it should be None. 

If so, `self.assertIsNotNone` and `self.assertIsNone` would be better to use

https://docs.python.org/3/library/unittest.html#unittest.TestCase.assertIsNotNone